pallets/parachain-staking/src/lib.rs

```diff
@@ -518,10 +517,15 @@ pub mod pallet {
 	pub(crate) type TotalSelected<T: Config> = StorageValue<_, u32, ValueQuery>;
 
 	#[pallet::storage]
-	#[pallet::getter(fn parachain_bond_info)]
-	/// Parachain bond config info { account, percent_of_inflation }
-	pub(crate) type ParachainBondInfo<T: Config> =
-		StorageValue<_, ParachainBondConfig<T::AccountId>, ValueQuery>;
+	#[pallet::getter(fn inflation_distribution_info)]
+	/// Inflation distribution configuration, including accounts that should receive inflation
+	/// before it is distributed to collators and delegators.
+	///
+	/// The sum of the distribution percents must be less than or equal to 100.
+	///
+	/// The first config is related to the parachain bond account, the second to the treasury account.
```

Comment:
This last comment line is wrong, the pallet staking doesn't care what theses accounts mean, it's juts to accounts that receive part of the inflation